wordpress-mobile/WordPress-Android
# Fix post signup showing reader overlay

This PR fixes an issue with the Post Signup navigation showing the user a JP overlay with no way to navigate out.

To recreate the issue
- Install the WP release or beta build
- Follow the signup flow
- On the Post Signup Interstitial tap the "Not right now" link
- ✅ Verify you are stuck on the JP reader overlay view

[post-signup-fix.webm](https://github.com/wordpress-mobile/WordPress-Android/assets/506707/4f07f627-6da2-43b5-aba4-e65f6c0c5209)

**To test:**
- Install the WP build
- Follow the signup flow
- On the Post Signup Interstitial tap the "Not right now" link
- ✅ Verify you are not stuck on the JP Reader overlay view

- 
## Regression Notes
1. Potential unintended areas of impact
The user remains stuck on the JP overlay reader view

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
Add unit tests to `PostSignupInterstitialViewModelTest`

PR submission checklist:

UI Changes testing checklist:
